### ISSUE TITLE
GitHub Search API Irregularity

### ISSUE THREAD

**Issue by gusbo010** [github/docs#17388]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/search-github/searching-on-github/searching-for-repositories

### What part(s) of the article would you like to see updated?

Remove the part in the linked article where it specifies that "**When you omit this qualifier, only the repository name and description are searched**". This is confirmed to be untrue since for example:
curl -s "https://api.github.com/search/repositories?q=graphql"
curl -s "https://api.github.com/search/repositories?q=graphql+in:name,description"
Yields different results.

A GitHub Support also confirmed that this is the case, at lest at the moment (presumably going forward as well).

The GitHub Support representative did not specify what fields were searched, just that it wasn't only name and description.
My own guess is that topics where also searched, but I am unable to fully confirm this.

TLDR; GitHub docks 'promises' behavior that is not followed. 

### Additional information

See: https://github.com/github/docs/discussions/17192 for more details

**Comment by gusbo010** [comment]
Working on PR

**Comment by Mk1206mk** [comment]
https://github.com/Mk1206mk/Mk1206mk.git
